documentation/docs/wp-creating-pages.md

```diff
@@ -0,0 +1,134 @@
+
+You can use Pages to organize and manage the structure of your website content. You can add as many pages to your site as you would like, and you can update your pages as many times as you want.
+
+After you create a page, you can add it to your site’s navigation menu so your visitors can find it.
+
+In general, Pages are very similar to Posts in that they both have titles and content. WordPress Theme template files maintain a consistent look throughout your site. Pages, though, have several key distinctions that make them different from Posts.
+
+**What Pages are**
+- Pages are for content that isn’t time-dependent, or which is not part of the blog content.
+- Pages can be organized into parent pages and subpages.
+- Pages can use different page templates, including template files, and Template Tags.
+- Some themes may provide different display options for individual Pages.
+- You can have a WordPress website with only Pages.
+
+**What Pages are not**
+- Pages don’t appear in the chronological view in the blog of a website.
+- Pages by default do not allow taxonomy (categories, tags, and any custom taxonomies). You can enhance it via plugins.
+- Pages are not files. They are stored in your database, just like Posts.
+- Pages are not included in your site’s feeds. (e.g. RSS or Atom.)
+- Pages and Posts can be interpreted differently by site visitors and by search engines. Search engines place more relevance on Posts because a newer post on a topic may be more relevant than a static page.
+- A specific page (or a specific post) can be set as a static front page. Websites that are set up this way have a blog page that displays the latest blog posts.
+
+## Page Screen
+
+To view all the Pages in your site:
+- Log in to your WordPress admin screen.
+- Click Pages in the left sidebar. This will show the Pages screen with a list of all the pages you have in your site.
+
+<img src="./md-assets/pages-admin-screen.png" hspace="30">
+
+1.	You can easily search for a Page based on the Page Title.
+2.	You can get a list of all the Published Pages or Draft Pages. 
+3.	You can filter Pages based on dates.
+4.	You can perform Bulk actions > Edit or Move to Trash by selecting multiple Pages from the list.
+5.	You can sort the Pages list based on Title, Date, Author etc by clicking the Header columns.
+
+## Organizing Pages
+
+You can organize your Pages into parent and child pages and create a hierarchy.
+
+For example, you can have an About Page as a parent page and create subpages under it for Careers, Board of Directors, Locations, Company Culture, Press etc. You can also have a Services Page as a parent page and create subpages for each of the services such as Web Design, Web Development, Stationery Design etc
+
+The structure of the pages on the site would then look like this.
+- About Us 
+- Board of Directors
+- Company Culture
+- Press
+- Careers
+- Locations
+- Services 
+- Web Design
+- Web Development
+- Stationery Design
```

Comment:
```suggestion
- About Us
	- Board of Directors
	- Company Culture
	- Press
	- Careers
	- Locations
- Services
	- Web Design
	- Web Development
	- Stationery Design
```
From the paragraph above about the structure of the pages it sounds like there should be a hierarchy that isn't coming through in the markdown output. This should fix it if I understand the hierarchy right.

Reply:
Good catch, thank you.